digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/assetUpdater/LinearAssetUpdater.scala

```diff
@@ -872,21 +872,16 @@ class LinearAssetUpdater(service: LinearAssetOperations) {
     
     if (changeSet.adjustedSideCodes.nonEmpty) {
       logger.debug(s"Saving SideCode adjustments for asset/link ids=${changeSet.adjustedSideCodes.map(a => s"${a.assetId}").mkString(", ")}")
-      adjustedSideCodes(changeSet.adjustedSideCodes)
+      dao.updateSideCodes(changeSet.adjustedSideCodes)
     }
-    
+
     if (changeSet.valueAdjustments.nonEmpty) {
-      logger.debug(s"Saving value adjustments for assets: ${changeSet.valueAdjustments.map(a => s"${a.asset.id}").mkString(", ")}")
+      logger.debug(s"Saving value adjustments for assets: ${changeSet.valueAdjustments.map(a => s"${a.assetId}").mkString(", ")}")
       changeSet.valueAdjustments.foreach { adjustment =>
-        service.updateWithoutTransaction(Seq(adjustment.asset.id), adjustment.asset.value.get, adjustment.asset.modifiedBy.get)
+        service.updateWithoutTransaction(Seq(adjustment.assetId), adjustment.value, adjustment.modifiedBy, fromAssetUpdater = true)
```

Comment:
tähän pitäisi saadaa joku massaoperaatio. Tää tulee olemaan erittäin hidas kohta koodissa. Jos tulee paljon päivityksiä. Kaikki hae yksitellen hommat pitäisi poistaa.

Reply:
Eikös nämä ollut vaan DamagedByThaw assetilla samuutuksessa? Siinä tapauksessa ei näitä paljoa tule

Reply:
Tätä käytetään myös leveyksissä ja päällysteissä nyt.

Reply:
Päällysteessä tulee siinä kohtaa, missä muokataan niin koneen luomia.

Reply:
Jeps, siinä tapauksessa massana

Reply:
Totutettu massapäivitys nyt silleen, että sen pitäisi selvitä leveydestä ja kelirikosta. Päällysteessä taitaa mennä tällä hetkellä loppujen lopuksi niin, että siinä vain poistetaan tai luodaan uusille linkeille, ei varsinaisesti muokata.